cloud/azure/bin/lib.sh

```diff
@@ -1 +1,11 @@
 #!/usr/bin/env bash
+
+readonly LIB_DIR="${BASH_SOURCE%/*}/lib"
+
+if [[ "${SOURCED_LIB}" != "true" ]]; then
```

Comment:
suggestion (blocking): there's a good chance this same pattern will be loaded in multiple places for a given script

```suggestion
if [[ "${SOURCED_AZURE_LIB}" != "true" ]]; then
```